app/models/perilune/trifle/stat.rb

```diff
@@ -0,0 +1,26 @@
+# frozen_string_literal: true
+
+module Commodity
+  module Trifle
+    class Stat
+      include Mongoid::Document
+
+      field :key
+      field :data
+      index({ key: 1 })
+
+      # custom Mongo configuration in addition to default config
+      # config/initializers/trifle.rb
+      def self.mongo_config
```

Comment:
Lets use Redis configuration here. Best would be to define this globally in an initializer. I'm not sure if that will play nicely with main app initializer.

Alternativelly you could devine these as a classes/options that user can specify in configuration. It's fairly expectable that app will use Redis, but Mongo is bit more specific. I would still give them option to just use Postgres if thats what they use.